### PR TITLE
added favicon to website

### PR DESCRIPTION
**Pull request by baldwmic**
**Summary**

- @flarnie @kedromelon finishes up #965 by adding the favicon to the website
- thanks to @kedromelon for the great favicon! 🎉 

#### Before
screen shot 2017-11-22 at 11 58 00

#### After
screen shot 2017-11-22 at 11 57 46

**Test Plan**

- not applicable

### CODE REVIEW

**Comment by kedromelon** [comment]
sweet!! thank you @baldwmic 😄

**Comment by baldwmic** [comment]
@juliankrispel @jackyho112 Great catch! We should definitely support other platforms.

- I used [Real Favicon Generator](https://realfavicongenerator.net/) like @juliankrispel  recommended and it was awesome! Simple to use and supports all browsers and platforms
- That site even has a favicon checker so once a new version of draft-js website is deployed we can check that each platform sees the appropriate favicon. Of course, currently the checker reports no favicons for [`draftjs.org`](https://realfavicongenerator.net/favicon_checker?protocol=http&site=draftjs.org#.WhmZJLQ-eL4)

Here you can see the website in development using the new favicon generated using the above method.
screen shot 2017-11-25 at 11 16 59

Thanks again to @kedromelon for the awesome favicon!

**Comment by baldwmic** [comment]
@juliankrispel friendly ping! have you seen the changes to this?

File: website/core/Site.js [hunk 1]
```diff
@@ -29,6 +29,14 @@ var Site = React.createClass({
...
           <meta property="og:description" content="Rich Text Editor Framework for React" />
 
+          <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png" />
+          <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png" />
+          <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png" />
+          <link rel="manifest" href="/img/manifest.json" />
+          <link rel="mask-icon" href="/img/safari-pinned-tab.svg" color="#5bbad5" />
+          <link rel="shortcut icon" href="/img/favicon.ico" />
+          <meta name="msapplication-config" content="/img/browserconfig.xml" />
+          <meta name="theme-color" content="#ffffff" />
```
Comment (juliankrispel) [inline website/core/Site.js]: just a tiny nitpick - would be nice if theme-color were the red background - `#852d2d`

**Comment by juliankrispel** [comment]
@baldwmic yes sry I just tend to do other things on the weekends ;)

File: website/core/Site.js (hunk 1 above)
Comment (baldwmic) [reply website/core/Site.js]: good catch, i will update to make the background-color red

**Comment by baldwmic** [comment]
@juliankrispel no worries, that's a good thing 😄 

i've updated the `theme-color` property to be the red background color

**Comment by baldwmic** [comment]
@flarnie when you get a chance to look at this, let me know if it's good to go!

**Comment by baldwmic** [comment]
@juliankrispel @flarnie @jackyho112 any update on this?

**Comment by flarnie** [comment]
Sweet - I'll get this landed. 🛬
Thanks @baldwmic and @juliankrispel @jackyho112 for reviewing. :)

**Comment by flarnie** [comment]
Also thanks @kedromelon for the favicon :)

**Comment by baldwmic** [comment]
awesome, thanks @flarnie looking forward to it 😄

**Comment by flarnie** [comment]
There are some errors with the scripts we use to import PRs, the FB maintainers will look into fixing it. Thanks for your patience everyone.

**Comment by flarnie** [comment]
Really happy that @zpao and @cdelahousse took the time to get this merged - thank you so much! :)

I don't see the new favicon showing up on draftjs.org - will try to look into it this weekend if it's still not showing up by then.

**Comment by baldwmic** [comment]
I see the favicon showing up!

screen shot 2018-03-23 at 12 04 20

**Comment by ddelrio1986** [comment]
Showing for me as well.

**Pull request closed without merging**